admin/base/settings/defaults.py

```diff
@@ -197,3 +198,23 @@
         ]
     },
 }
+
+# Keen.io settings in local.py
+KEEN_PROJECT_ID = 'CHANGEME'
+KEEN_READ_KEY = 'CHANGEME'
+KEEN_WRITE_KEY = 'CHANGEME'
```

Comment:
right now osf settings only have KEEN_PROJECT_ID and KEEN_WRITE_KEY defined. I'll go ahead and add KEEN_READ_KEY as well then point the settings to osf_settings